core/src/test/java/cucumber/runtime/TestHelper.java

```diff
@@ -189,16 +196,18 @@ public Glue get() {
             }
         };
 
-        Runtime.builder()
```

Comment:
Because we're mocking the time stamps here the bus has to be passed directly into the runner (there can't be a hierarchy of bussses here).  This can't be done in a pretty way by the builder. 

That said, the callers of this method could stand to have a stern looking at. There is too much setup cruft in those tests.

Reply:
Yeah I find a lot of tests being confusing, mainly in how they are setting things up e.g. glue, hooks etc....  plus soooo many overloads to call into actually running the code.

Some of the asserting is a little messy, would like to use assertJ core and a nice json assert library, which would make things more readable.